mlrun/db/base.py

```diff
@@ -242,7 +242,7 @@ def _resolve_artifacts_to_tag_objects(
             )
             artifact_identifiers.append(
                 mlrun.common.schemas.ArtifactIdentifier(
-                    key=mlrun.utils.get_in_artifact(artifact_obj, "key"),
+                    key=mlrun.utils.get_in_artifact(artifact_obj, "db_key"),
```

Comment:
worth a comment on code why do you extract the db_key and not the key or even (logged by project vs logged by run)